src/imageTools/saveAs.js

```diff
@@ -1,6 +1,8 @@
 import { $ } from '../externalModules.js';
 
-export default function (element, filename) {
+export default function (element, filename, mimetype) {
```

Comment:
Can you make "image/png" the default mimetype?

Reply:
I made it the *old way*:

    // setting the default value for mimetype to image/png
    mimetype = mimetype || "image/png";

Of course I can change to `param = defaultvalue`, do you prefer that way?.

Reply:
It's a minor difference, I guess there's no need to change it. I realized it didn't matter and just merged it :-) Thanks!